DESCRIPTION

```diff
@@ -5,7 +5,7 @@ Title: The Maraca Plot: Visualization of Hierarchical Composite Endpoints in Cli
 License: Apache License (>= 2)
 Authors@R: c(
     person("Martin Karpefors", "", email = "martin.karpefors@astrazeneca.com", role = "aut"),
-    person("Samvel Gasparyan", "", email = "samvel.gasparyan@astrazeneca.com", role = "aut"),
+    person("Samvel B. Gasparyan", "", email = "samvel.gasparyan@astrazeneca.com", role = "aut"),
     person("Stefano Borini", "", email = "stefano.borini@astrazeneca.com", role = c("ctb", "cre")))
 Description: Library that supports visual interpretation of hierarchical composite
     endpoints (HCEs). HCEs are complex constructs used as primary endpoints in
```

Comment:
You have to remove the sanon dependency and add hce in the DESCRIPTION file as well. Keep into account that, while roo honors less-than constraints, R installation tools don't (and adding them may actually fail. Not sure. There's an open bug I've seen). So you will have to specify only hce (>= 0.0.2).